### PR TITLE
Style Progress Bar and a few minor tweaks 

### PR DESCRIPTION
**Pull request by FredHappyface**
Thank you for an awesome project - was a great starting point for some dev I was doing 

This PR adds the following:

- add dark scrollbar for dark theme
- consistent hovering (brightness is applied to navbar hovering)
- style progress bar

(Adapted from changes for some webdev I was doing - https://fhpwa.github.io/. There were some other changes made around theming but I've held off as I'm not sure if that diverges too much from the theming this project aims to do. On the off-chance it would be useful happy to contribute further)

Hope this is useful, any tweaks etc - please let me know

### CODE REVIEW

**Comment by kevquirk** [comment]
Hey, thanks for this - lots of tidy up that just becomes white noise as I spend so much time in this file. I think Atom automatically cleans up the tabbing, so I don't see that. So appreciate the tidying there.

I don't like the colour changes though - using `accent-bg` in place of `accent` for things like buttons isn't what I want, and it would be a breaking change for the sites that use simple.css.

If you remove the changes to the colours, I'll merge the rest.

Thanks,

Kev

File: simple.css [hunk 1]
```diff
@@ -252,8 +254,8 @@ header > nav a:visited {
 }
 
 header > nav a:hover {
```
Comment (FredHappyface) [inline simple.css:256]: Hopefully addressed your comments. Would you like this reverting too? This was included to provide consistency with other hover items like buttons

**Comment by FredHappyface** [comment]
Hi just dropping a follow up comment here. Happy to make additional changes if needed :)

**Comment by kevquirk** [comment]
Thanks for the prompt - been meaning to re-visit this.

**Pull request merged**